app/controllers/api/integrations_controller.rb

```diff
@@ -69,9 +69,9 @@ def update_production
   def update_onpremises_production
     if @proxy.update_attributes(proxy_params)
       flash[:notice] = flash_message(:update_onpremises_production_success)
-      redirect_to action: :edit, anchor: 'production'
+      redirect_to action: :show, anchor: 'production'
     else
-      render :edit
+      render :show
     end
   end
```

Comment:
Please do not change this method in this PR and leave it for https://github.com/3scale/porta/pull/2288